package-lock.json

```diff
@@ -3004,17 +3004,15 @@
       "version": "file:packages/components",
       "dev": true,
       "requires": {
-        "@jobber/design": "^0.29.2",
-        "@jobber/formatters": "^0.2.0",
-        "@jobber/hooks": "^1.12.4",
```

Comment:
Why are these removed? Is this lockfile out of date from a previous merge? Maybe an atlantian can answer that.

Also, there are a lot of hash changes committed below (the ones that go from sha512 to sha1). I would not commit those.

Reply:
combination of lerna and auto-resolved merge conflicts with package-lock since we do lots of package upgrades left and right. Honestly, if lerna removes them (`npm ci` on root) I wouldn't worry too much on it.

I'll test on my machine tho

Reply:
Please review again! Updated a version on the package-lock file